File: elearning-service/src/main/java/com/app/elearningservice/dto/ResultDTO.java
package com.app.elearningservice.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ResultDTO {
    private Long resultId;
    private Long quizId;
    private String quizTitle;
    private String scoreStr;
    private String status;
    private String startTime;
    private String endTime;
}

File: elearning-ui/src/app/shared/model/ResultDTO.ts
export class ResultDTO {
  constructor(
    public resultId: number = 0,
    public quizId: number = 0,
    public quizTitle: string = '',
    public scoreStr: string = '',
    public status: string = '',
    public startTime: string = '',
    public endTime: string = '',
  ) {
  }
}
